stdlib/websub/src/main/ballerina/websub/Module.md

```diff
@@ -10,40 +10,86 @@ This implementation supports introducing all WebSub components: subscribers, pub
 is updated by the topic's publisher.
  
  
-#### Basic Flow with WebSub
+### Basic Flow with WebSub
 1. Subscriber discovers the topics it needs to subscribe to in order to receive updates/content and discovers the hub(s) where it can subscribe.
 
 2. Subscriber sends a subscription request to a hub, specifying the topic it needs to receive notifications for along 
  with other subscription parameters, such as:
-      - The callback URL where content is expected to be delivered
-      - (Optional) A lease seconds value indicating how long the subscriber wants the subscription to stay active
-      - (Optional) A secret to use for [authenticated content distribution](https://www.w3.org/TR/websub/#signing-content)
+    - The callback URL where content is expected to be delivered
+    - (Optional) A lease seconds value indicating how long the subscriber wants the subscription to stay active
+    - (Optional) A secret to use for [authenticated content distribution](https://www.w3.org/TR/websub/#signing-content)
   
-3. The hub sends an intent verification request to the specified callback URL, and if the response indicates verification
+3. The hub sends an intent verification request to the specified callback URL, and if the response indicates 
+verification
  (by echoing a challenge specified in the request) by the subscriber, the subscription is added for the topic at the 
  hub.
    
 4. Publisher notifies the hub of updates to the topic, and the content to deliver is identified.
 
 5. The hub delivers the identified content to the subscribers of the topic.
 
-#### Features
+### Features
+
+#### Subscriber
+
 This module allows introducing a WebSub Subscriber Service with `onIntentVerification`, which accepts HTTP GET requests for intent verification, and `onNotification`, which accepts HTTP POST requests for notifications. The WebSub Subscriber Service provides the following capabilities:
  - Subscription Requests are sent at service start time for the hub and topic, which are either specified as annotations or discovered based on the resource URL specified as an annotation.
```

Comment:
```suggestion
 - Subscription Requests are sent at the start time of the service for the hub and topic, which are either specified as annotations or discovered based on the resource URL specified as an annotation.
```

Reply:
Will change to " When the service is started a subscription request is sent for a hub/topic combination, either specified as annotations or discovered based on the resource URL specified as an annotation."